gateway-service-definitions/src/main/resources/services/hdfsui/2.7.0/service.xml

```diff
@@ -16,6 +16,12 @@
    limitations under the License.
 -->
 <service role="HDFSUI" name="hdfs" version="2.7.0">
+    <metadata>
+        <type>UI</type>
+        <context>/hdfs?host={{BACKEND_HOST}}</context>
```

Comment:
There is no `?host=` parameter on the 2.7.0 version of the HDFS UI service definition.

Reply:
Fixed.